src/state/megaStore.tsx

```diff
@@ -224,6 +224,10 @@ export const Provider: ParentComponent = (props) => {
         }
     };
 
+    onCleanup(() => {
+        console.warn('Parent Component is being unmounted!!!');
```

Comment:
Maybe should put the current timestamp of this into local storage or something so we can see if it happened on a mobile app

Reply:
I don't think we do a export of all the data that might be in storage, just from what is relevant to mutiny-node. We could create something for the frontend to log and export but need to build that out.

Reply:
oh yeah it wont

Reply:
made an issue for this would be good for sluthing: https://github.com/MutinyWallet/mutiny-web/issues/271